tests/models/bart/test_modeling_bart.py

```diff
@@ -1516,3 +1516,7 @@ def test_retain_grad_hidden_states_attentions(self):
 
     def test_save_load_fast_init_from_base(self):
         pass
+
+    @unittest.skip("The model doesn't support left padding")  # and it's not used enough to be worth fixing :)
```

Comment:
Note that this is not Bart, but the BartDecoder used as a stand-alone model. 13 out of the 16 skips below follow this pattern, a decoder used as stand-alone